api/nakadi-event-bus-api.yaml

```diff
@@ -548,6 +548,38 @@ paths:
           description: |
             Schema object.
 
+  '/event-types/{name}/cursor-distances':
+    get:
+      tags:
+        - unmanaged-api
+        - monitoring
+        - management-api
+      security:
+        - oauth2: ['nakadi.event_stream.read']
+      description: |
+        Calculate the distance between two offsets. This is useful for performing checks for data completeness, when
+        a client has read some batches and wants to be sure that all delivered events have been correctly processed.
+      parameters:
+        - name: X-nakadi-cursor-distances-query
+          in: header
```

Comment:
I'm not that sure about that, because a subscription listener in best case wants to apply this operation on all partitions he is listening to at once. Surely it is an option to provide two end point for this request, but I would understand the decision to only support one bulk API, as it is already complex and big enough.